### PR TITLE
CI: Store test metrics

### PR DESCRIPTION
**Pull request by gavin-stackrox**
## Description

Per title. @janisz I used v0.0.8 as I wasn't sure about the extra 0 in the v0.0.07, typo? or?...

## Checklist
- [x] Investigated and inspected CI test results

## Testing Performed

CI is sufficient. Should see lots of test results in BQ.

### CODE REVIEW

File: scripts/ci/lib.sh [hunk 1]
```diff
@@ -1226,12 +1226,14 @@ store_test_results() {
...
     local from="$1"
     local to="$2"
+    local csv_output
 
     set +u
     if ! is_in_PR_context; then
     {
         info "Creating JIRA task for failures found in $from"
-        curl --retry 5 -SsfL https://github.com/stackrox/junit2jira/releases/download/v0.0.6/junit2jira -o junit2jira && \
+        csv_output="$(mktemp --suffix=csv)"
```
Comment (janisz) [inline scripts/ci/lib.sh:1235]:
```suggestion
        csv_output="$(mktemp --suffix=.csv)"
```

**Review by janisz — APPROVED** [review]
LGTM

Thanks for handling my typo and creating new release!

**Comment by janisz** [comment]
Do you think we should also gather unit tests stats? 
 https://github.com/stackrox/stackrox/blob/master/.github/actions/junit2jira/action.yaml#L32

**Comment by gavin-stackrox** [comment]
> Do you think we should also gather unit tests stats? https://github.com/stackrox/stackrox/blob/master/.github/actions/junit2jira/action.yaml#L32

Yeah, I think it would be useful. I didn't want to tackle it yet though because it'll need a GCP login and it doesn't feel right to just hack that in to the `junit2jira` action without a bit of refactoring.

**Pull request merged**